src/test/regress/expected/multi_insert_select_non_pushable_queries.out

```diff
@@ -684,7 +684,7 @@ $Q$);
  Custom Scan (Citus INSERT ... SELECT)
    INSERT/SELECT method: pull to coordinator
    ->  HashAggregate
-         Group Key: remote_scan.user_id, remote_scan.event_type
+         Group Key: remote_scan.user_id, remote_scan.value_1_agg
```

Comment:
Now the plan matches the plan of the actual query sent to workers. We modify the SELECT query by renaming columns & possibly wrapping it before sending it to workers.